components/doc/carousel/index.js

```diff
@@ -549,6 +549,24 @@ const responsiveOptions = [
                                     <td>false</td>
                                     <td>Defines if scrolling would be infinite.</td>
                                 </tr>
+                                <tr>
```

Comment:
Also please update `\api-generator\components\carousel.js`

Reply:
> Also please update `\api-generator\components\carousel.js`
Thank you for your kind reminder @melloware. I added the necessary attributes to the file as you specified.